disqus/disqus.php

```diff
@@ -510,6 +510,7 @@ function dsq_request_handler() {
                         die('// synced '.$comments.' comments');
                     }
                 } else {
+                    wp_clear_scheduled_hook( 'dsq_sync_forum' );
                     $ts = time() + 300;
```

Comment:
this may not work. Rather https://github.com/disqus/disqus-wordpress/commit/ba0a89ef21e74849c7138052696b4c02c857b72c is better solution.